frontend/app/components/Spots/SpotPlayer/components/SpotVideoContainer.tsx

```diff
@@ -164,17 +164,27 @@ function SpotVideoContainer({
   return (
     <>
       {processingMsg ? (
-        <div>This video is being processed</div>
+        <div className='relative w-full h-full flex flex-col items-center justify-center bg-white/50'>
+          <img
+            src={'../assets/img/videoProcessing.svg'}
+            alt={'Optimizing video..'}
+            width={75}
+            className="mb-5"
+          />
+          <div className={'text-2xl font-medium '}>
+            Optimizing Spot Recording
```

Comment:
its not optimizing, its processing -- uploading to backend and generating s3 link. Anyways, if you  want to use exact same component as we're using for loading here, why copy pasting it instead of adding a case to `{!isLoaded &&` -> `{!isLoaded || processingMsg ? (component) : null`